### PR TITLE
Data: RishikaVishnoi

### PR DESCRIPTION
**Pull request by Rishikavishnoi**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5182"></a>

### CODE REVIEW

File: data/Rishikavishnoi.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+
+	"name": "Rishika Vishnoi",
+
+	"displayStatsPublic": true,
```
Comment (Pradumnasaraf) [inline data/Rishikavishnoi.json:5]:
```suggestion
```

File: data/Rishikavishnoi.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+	"type": "personal",
+
+	"bio": "Full Stack Developer | JavaScript | React | Open Source Enthusiast",
+
+	"avatar": "https://github.com/rishikavishnoi.png",
```
Comment (Pradumnasaraf) [inline data/Rishikavishnoi.json:11]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

In the latest release, both of property is not required. So, please remove them

**Comment by Rishikavishnoi** [comment]
@Pradumnasaraf I have removed it. Kindly Review  Now

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**